### PR TITLE
fix(FEC-8588): large play button displayed for a sec before the next item is playing

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

use `autoplay` config for the second entry and onwards 

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by yairans** [comment]
@dan-ziv the large play button just raised the issue, but the solution is correct any way. play next means autoplay true.

**Comment by dan-ziv** [comment]
@yairans No, play next means **autoContinue** true. The autoplay is separate configuration which is not related to the playlist (In playlist context it effected only the first entry). There's no issue here from the player side.

**Comment by yairans** [comment]
@dan-ziv the core player doesn't know about autoContinue config and playlist flow at all.
from its side play next means load an entry and play it automatically

**Comment by dan-ziv** [comment]
The core player isn't related to this. The kaltura player manage the playlist logic and in the same way that the core play manage autoplay (by internally calling play()) the kaltura player should manage autoContinue. The above issue is a pure UI issue.

File: src/common/playlist/playlist-manager.js [hunk 1]
```diff
@@ -34,14 +36,18 @@ class PlaylistManager {
         this._player.dispatchEvent(new FakeEvent(PlaylistEventType.PLAYLIST_LOADED, {playlist: this._playlist}));
         const next = this._playlist.next;
         if (next.item) {
-          this._setItem(next.item, next.index);
+          this._setItem(next.item, next.index).then(() => {
+            this._eventManager.listenOnce(this._player, PlaylistEventType.PLAYLIST_ITEM_CHANGED, () => {
+              this._player.configure({playback: {autoplay: this._options.autoContinue}});
```
Comment (OrenMe) [inline src/common/playlist/playlist-manager.js]: wouldn't that cause each call to configure to do this? so calling configure multiple times will cause this config to fire.

Comment (yairans) [reply src/common/playlist/playlist-manager.js]: fixed. 10x.
new it's only when new sources configured

**Comment by yairans** [comment]
@dan-ziv as approved by @itaykinnrot and @OrenMe any play next flow (even by user gesture) consider as auto play. So this fix is correct also without the UI issue.

**Pull request merged**